core/controllers/questions_list.py

```diff
@@ -42,15 +42,30 @@ class QuestionsListHandler(base.BaseHandler):
     """
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'comma_separated_skill_ids': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'offset': {
+                'schema': {
+                    'type': 'int'
+                },
+                'default_none': None
+            }
+        }
+    }
 
     @acl_decorators.open_access
     def get(self, comma_separated_skill_ids):
         """Handles GET requests."""
         try:
-            offset = int(self.request.get('offset'))
-        except Exception as e:
-            raise self.InvalidInputException('Invalid offset') from e
-
+            offset = (self.normalized_request.get('offset'))
```

Comment:
```suggestion
            offset = self.normalized_request.get('offset')
```

Wrapping parentheses not necessary.

Reply:
okay, thanks for update